src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -168,8 +171,10 @@ public void validate(UIValidationContext context)
          // Do not validate again if next() was called
          return;
       }
+
+      List<String> step = (List<String>) attributeMap.get(RETRY_STEP);
       DeploymentType deploymentType = (DeploymentType) attributeMap.get(DeploymentType.class);
-      if (deploymentType == DeploymentType.CD)
+      if (deploymentType == DeploymentType.CD && (step == null || "0".equals(step.get(0))))
```

Comment:
Is this condition necessary?

Reply:
yes, otherwise you can't use the retry as the validation will say that things already exist.